public/ACMW/css/events.css

```diff
@@ -62,61 +53,33 @@
 }
 .open .dropdown-toggle {
 	color: #fff;
-	background-color: #555;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

Reply:
here also we have to use it

Reply:
Have a look at this: [Disallow !important](https://github.com/CSSLint/csslint/wiki/Disallow-!important)